src/main/kotlin/app/revanced/patches/youtube/misc/fix/verticalscroll/fingerprints/CanScrollVerticallyFingerprint.kt

```diff
@@ -7,6 +7,17 @@ import org.jf.dexlib2.Opcode
 object CanScrollVerticallyFingerprint : MethodFingerprint(
     "Z",
     parameters = emptyList(),
-    opcodes = listOf(Opcode.INSTANCE_OF),
+    opcodes = listOf(
+        Opcode.IGET_OBJECT,
+        Opcode.INSTANCE_OF,
+        Opcode.CONST_4,
+        Opcode.IF_EQZ,
+        Opcode.CHECK_CAST,
+        Opcode.INVOKE_STATIC,
+        Opcode.MOVE_RESULT,
+        Opcode.RETURN,
+        Opcode.INVOKE_VIRTUAL,
+        Opcode.MOVE_RESULT,
```

Comment:
Wouldn't these two suffice? The pattern is longer than necessary.

Reply:
It is recommended, because I need them to get the registry value and the index where place the patch.

Reply:
EDIT: ok...I reduced the fingerprint, and patch still works...also if I'm not 100% sure it is a good idea (because a too small a fingerprint could cause problems in the future imho).

Reply:
The class for this method is constrained by `customFingerprint`. I believe this would suffice for this fingerprint.

Reply:
> The class for this method is constrained by `customFingerprint`. I believe this would suffice for this fingerprint.

Ok...done